korman/properties/modifiers/sound.py

```diff
@@ -152,6 +152,16 @@ def _update_name(self, context=None):
                          options={"ANIMATABLE"},
                          subtype="PERCENTAGE")
 
+    use_reverb = BoolProperty(name="Reverberation",
+                              description="Allow the sound to be affected by reverberation regions",
+                              default=False,
+                              options=set())
```

Comment:
I feel like this should probably be implicit. We almost definitely want all 3D sounds (read: sound effects) to be affected by environmental audio, so that's an extra button press required for them. Whereas background music, GUI sounds, ambiance, etc. probably don't want reverb. If someone wants to explicitly disable reverb on a sound effect, we could test for the `reverb_amount == 0` and just toggle it off.

Reply:
Makes sense.